Gems/PhysX/Code/NumericalMethods/Include/NumericalMethods/DoublePrecisionMath/Quaternion.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+#include <AzCore/Math/Quaternion.h>
+
+namespace NumericalMethods::DoublePrecisionMath
+{
+    class Quaternion
```

Comment:
would it be possible to define a `Quaterniond` in AZCore math?

Reply:
AFAIK the only place double precision math is used so far is here and for Vector3 in the primitive shape fitting which is part of the PhysX pipeline. Both of those only define the minimum functionality for those use cases rather than a complete API. 

If there are enough use cases for double precision math, it makes sense to add those types with complete APIs in AzCore/math, but I don't think it's in scope for this PR.